index.js

```diff
@@ -364,6 +399,16 @@ function initApp(callback) {
     }
   });
 
+  startDnsServers = new Promise((fulfill, reject) => {
```

Comment:
Why is there no equivalent `stopDnsServers` ? Shouldn't there be a shutdown hook that stops these processes?